src/amo/components/FeaturedCollectionCard/index.js

```diff
@@ -41,7 +41,7 @@ export default class FeaturedCollectionCard extends React.Component<Props> {
         })}
         header={header}
         footerText={footerText}
-        footerLink={`/collections/${username}/${slug}/`}
+        footerLink={`/collections/${userId}/${slug}/`}
```

Comment:
This is certainly not something to address in this patch (if at all), but I have to say I was quite confused to see that this passes in a `footerLink`, but the link is actually displayed in the header and not the footer. I suppose at some point it was in the footer of the card and it was moved into the header by means of css without renaming anything. :-P

Reply:
That's because we are mobile-first from Day 1. This card renders both a header and a footer on small screens. On larger screens, the footer is hidden and moved to the header, hence the `footerLink` in the header.

I am not saying it is a good thing though 😛